README.md

```diff
@@ -34,6 +34,7 @@ Portions Copyright (c) 2012-2013 The Cryptonote developers.
 - Mail: [dev@getmonero.org](mailto:dev@getmonero.org)
 - GitHub: [https://github.com/monero-project/monero](https://github.com/monero-project/monero)
 - IRC: [#monero-dev on Freenode](https://webchat.freenode.net/?randomnick=1&channels=%23monero-dev&prompt=1&uio=d4)
+* It is HIGHLY recommended that you join the #monero-dev IRC channel if you are developing software that uses Monero. Due to the nature of this open source software project, joining this channel and idling is the best way to stay updated on best practices and new developments in the Monero ecosystem. All you need to do is join the IRC channel and idle to stay updated with the latest in Monero development. If you do not, you risk wasting resources on developing integrations that are not compatible with the Monero network. The Monero core team and community continuously make efforts to communicate updates, developments, and documentation via other platforms – but for the best information, you need to talk to other Monero developers, and they are on IRC. #monero-dev is about Monero development, not getting help about using Monero, or help about development of other software, including yours, unless it also pertains to Monero code itself. For these cases, checkout #monero. 
```

Comment:
Maybe worth adding that the chat is not only on IRC, but it's also relayed on MatterMost and Matrix

Reply:
feel free to make a commit on top of this one, or however the gits allow it. I for one won't encourage these ghost systems that prevent me from tab autocompleting someones name, and are otherwise a pain to moderate and also depend on relay infrastructure that may or may not be working etc.

Reply:
I think it's important to make people know that they can join using multiple systems, this will increase the possibility to see a dev actually joining, but i see your point. Fair enough.